crates/sui-framework/packages/deepbook/sources/math.move

```diff
@@ -3,8 +3,8 @@
 
 module deepbook::math {
     /// scaling setting for float
-    const FLOAT_SCALING: u64 = 1000000000;
-    const FLOAT_SCALING_U128: u128 = 1000000000;
+    const FLOAT_SCALING: u64 = 1_000_000_000;
```

Comment:
Perhaps delete this now that it is unused?

Reply:
OK